improver_tests/calibration/reliability_calibration/test_ApplyReliabilityCalibration.py

```diff
@@ -230,6 +230,143 @@ def test_exception_without_relative_to_threshold(self):
             self.plugin._ensure_monotonicity(self.forecast)
 
 
+class Test__combine_undersampled_bins(unittest.TestCase):
+
+    """Test the _combine_undersampled_bins method."""
+
+    def test_no_undersampled_bins(self):
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=6014128)